src/github/issue-comment.ts

```diff
@@ -18,7 +18,7 @@ export const issueCommentWebhookHandler = async (
 	} = context.payload;
 	let linkifiedBody;
 
-	const gitHubInstallationClient = await createInstallationClient(githubInstallationId, jiraClient.baseURL, context.log);
+	const gitHubInstallationClient = await createInstallationClient(githubInstallationId, context.log, jiraClient.baseURL);
```

Comment:
Why the signature change?

Reply:
This was something I mentioned in the previous PR: I was originally using jiraHost to check for a github app but that's changed now since we've updated the models. We're literally just passing the jiraHost in so we can check if the FF is on or not. Once the FF is removed, all the jiraHost args will be deleted. Figured it made more sense to move the temporary param in the function definition to the end.